README.markdown

```diff
@@ -420,10 +420,12 @@ Use self only when required by the compiler (in `@escaping` closures, or in init
 
 For conciseness, if a computed property is read-only, omit the get clause. The get clause is required only when a set clause is provided.
 
+For single-expression return values where the context is clear, use implicit returns:
+
 **Preferred**:
 ```swift
 var diameter: Double {
-    return radius * 2
+    radius * 2
```

Comment:
I'm actually not a huge fan of omitting return. Don't feel super strongly though so curious what others think.

Reply:
I don't really like leaving off the return either. I think I can be convinced if using it in some cases but in general I prefer the return: https://onuniverse.slack.com/archives/CFJR85JTE/p1589898451216200?thread_ts=1589897258.215800&cid=CFJR85JTE

Reply:
I can be swayed either way. But, I echo Peter's point about having to refer to the function signature to determine what is/should be returned is an extra step that isn't necessary when return is present.

Reply:
I am happy to always use `return` in funcs and computed properties like we have always done. 

But what is the difference between this and [line 557](https://github.com/onuniverse/swift-style-guide/pull/8/files#diff-0a369498a5a8db3ac8fa606b544c9810L557)

I think that because `array.map { $0 * 2 }` vs. `array.map { return $0 * 2 }` has been there since like swift 2, we got used to it.

Reply:
I think it makes sense for SwiftUI, but tends to feels weird elsewhere (old habits die hard). I think we should keep explicit `return`s unless writing SwiftUI, or until the SwiftUI "meta" becomes more commonplace within our codebase.

Victor makes a good point tho, it's extremely common to omit the `return`s when using functional methods like `.map`, and def think we should continue to "allow" that

Reply:
I think another difference is you can't do a map without returning. A function on the other hand can existing without returning. So that extra assumption for the function makes you have to think and check the function signature. But that assumption for a map is correct and requires no extra thought.

Reply:
I wonder if this will feel different once we're able to use more Combine. On the backend where we have a lot of small chaining events and nested callbacks because of promises these returns start to make it harder to read instead of easier (Peter also talked me off this ledge previously, so we still have explicit returns on the API side, but I keep thinking about it)